### ISSUE TITLE
Enable master branch protection

### ISSUE THREAD

**Issue by derps-airmash** [steamroller-airmash/airmash-server#106]
Would prefer to wait to merge rather than accidentally break CI. What do you think?

Experience from $other projects suggests I'm too impulsive to manage a reliably working master without assistance :)

**Comment by steamroller-airmash** [comment]
A prerequisite for this is probably to set up working CI and to actually setup some tests of the server.

I'm not sure how to properly do feature/integration tests of the server but a simple CI that checks everything compiles and runs the unit tests is doable.

Perhaps we should migrate this repo to airmash-refugees and setup the infrastructure there?

**Comment by derps-airmash** [comment]
I'm ambivalent about putting your nice clean roomesque effort in the same GitHub org as the copyright frontend. Really doubt anything bad will happen, but one DMCA might be enough to make your repo unavailable forever.

On the other hand, having a single GitHub org for everything really helps show the project is not dead, and give some sense of identity that others might want to get involved with.

Your call :) Quite happy either way. (Also quite happy if the name of the org/avatar/etc are bikeshedded). The -eu fork that's already in there could also be renamed and just remain a perpetual mirror of your real repo, where I could work from a branch. It might afford some protection in the case of a DMCA request

**Comment by steamroller-airmash** [comment]
I think having a single github org for everything outweighs the potential downsides. In addition, we can keep my copy as an up-to-date mirror which should hopefully protect against DMCA requests.

Aside: IANAL and I don't know much about the DMCA laws but aren't DMCA requests restricted to the repo that would actually be infringing?

**Comment by steamroller-airmash** [comment]
I've enabled branch protection for the master branch. From now on everything needs to be approved before it can be merged into master.
